_episodes/03-create.md

```diff
@@ -44,21 +47,36 @@ $ cd planets
 {: .language-bash}
 
 Then we tell Git to make `planets` a [repository]({{ page.root }}{% link reference.md %}#repository)
--- a place where Git can store versions of our files:
+-- a place where Git can store versions of our files.
 
+It is good practice to first check if this is a good place to do so:
+a new repository shall not be nested in another repository placed higher up the directory hierarchy.
```

Comment:
this sentence introduces topic of nested repositories. Learners will ask, "what does that mean" and "why shouldn't we do that" which is explained later in the lesson.